starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,46 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
-<head>
-  <title>Apple Pie Recipe</title>
-</head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <head>
+    <title>Apple Pie Recipe</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="css/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet">
+  </head>
+  <body>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+    <header id="hero">
+      <h1>Apple Pie</h1>
+    </header>
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
-
-Place the bottom crust in your pan. Fill with apples, mounded slightly. Cover with a
-lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-that it does not run off.
-
-Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
-degrees C). Continue baking for 35 to 45 minutes, until apples are soft.
-</body>
-</html>
+    <section id="description" class="container">
+      <h3>This was my grandmother's apple pie recipe. I have never seen another one</h3>
+      <h3>quite like it. It will always be my favorite and has won me several first place</h3>
+      <h3>prizes in local competitions. I hope it becomes one of your favorites as well!</h3>
```

Comment:
Great looking HTML!! On my screen in specific there seems to be an issue with this area right here(I have a low resolution screen). Because they are h3, they are too big so they collapse onto each other. I also wouldn't advise to put text like this inside h3. It's still a paragraph of text so a p would be better.